migrations/2024-11-19-142005_instance-block-mod-log/up.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE federation_blocklist
+    ADD COLUMN expires timestamptz;
+
+CREATE TABLE admin_block_instance (
+    id serial PRIMARY KEY,
+    instance_id int NOT NULL REFERENCES instance (id) ON UPDATE CASCADE ON DELETE CASCADE,
+    admin_person_id int NOT NULL REFERENCES person (id) ON UPDATE CASCADE ON DELETE CASCADE,
+    blocked bool NOT NULL,
+    reason text,
+    expires timestamptz,
+    published timestamptz NOT NULL DEFAULT now()
```

Comment:
All the mod tables use a `when_` column instead of published. We should pry keep that uniform for now.